sklearn/kernel_approximation.py

```diff
@@ -358,6 +329,156 @@ def _transform_sparse(self, X):
         return sp.hstack(X_new)
 
 
+class AdditiveChi2Sampler(BaseAdditiveHomogenousKernelSampler):
+    """Approximate feature map for additive chi2 kernel.
+
+    Uses sampling the fourier transform of the kernel characteristic
+    at regular intervals.
+
+    Since the kernel that is to be approximated is additive, the components of
+    the input vectors can be treated separately.  Each entry in the original
+    space is transformed into 2*sample_steps+1 features, where sample_steps is
+    a parameter of the method. Typical values of sample_steps include 1, 2 and
+    3.
+
+    Optimal choices for the sampling interval for certain data ranges can be
+    computed (see the reference). The default values should be reasonable.
+
+    Read more in the :ref:`User Guide <additive_chi_kernel_approx>`.
+
+    Parameters
+    ----------
+    sample_steps : int, optional
+        Gives the number of (complex) sampling points.
+    sample_interval : float, optional
```

Comment:
It is just to be consistent through the file (see [RBFSampler](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/kernel_approximation.py#L24), [SkewedChi2Sampler](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/kernel_approximation.py#L112), [Nystroem](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/kernel_approximation.py#L361)).